thrift/shadower.thrift

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2017-2021 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+namespace java com.uber.cadence.shadower
+
+include "shared.thrift"
+
+const string ShadowerLocalDomainName = "cadence-shadower"
+const string ShadowerTaskList = "cadence-shadower-tl"
+
+const string ShadowWorkflowName = "cadence-shadow-workflow"
```

Comment:
Will this be used as a prefix of the workflow name?

Reply:
Good call. I forgot to add workflow ID suffix in the definition.

I am thinking `ShadowWorkflowName`, is the workflow name used by the worker to start the workflow. And probably `ShadowWorkflowIDSuffix = "-shadow-workflow"` is the suffix we add after domain name for the workflowID, so the workflowID will look like `<domainName>-shadow-workflow`. Thoughts?

Reply:
Sounds good to me